Tests/ImageTintingTests.swift

```diff
@@ -84,5 +86,23 @@ class ImageTintingTests: XCTestCase {
 
         BONAssertEqualImages(untintedResult!, tintAttemptResult!)
     }
+    
+    func testAccessibilityIOS() {
+        #if os(iOS)
```

Comment:
I initially had it in one method but decided it made the code harder to read. Updated to include tvOS.